themes/docsy/layouts/practices/single.html

```diff
@@ -0,0 +1,121 @@
+            <div class="row">
+                <div class="col-lg-4 px-2 px-lg-2 mb-1">
+                    {{ partial "practices/info-box" (dict "context" . "title" "Why do it?" "info" .Params.why ) }}
+                </div>
+                <div class="col-lg-4 px-2 px-lg-2 mb-1">
+                    {{ partial "practices/info-box" (dict "context" . "title" "When to do it?" "info" .Params.when ) }}
+                </div>
+                <div class="col-lg-4 px-2 px-lg-2 mb-1">
+                    {{ partial "practices/info-box" (dict "context" . "title" "What supplies are needed?" "info" .Params.what ) }}
+                </div>
+            </div>
+            {{ if .Params.remote }}
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-4 px-2 px-lg-2 mb-3">
+                        <h4 class="mt-3">How to use this method remotely?</h4>
+                        <p>There is currently a remote template available in Miro. Here you will find step
+                            instructions on how to conduct this practice remotely including general tips and
+                            info.
+                            <br/><br/>
+                            Disclaimer: In case you are not able to use Miro during your current
+                            engagement, please open the template using your Miro VMware account, take a
+                            print
+                            screen and copy it over to the tool you are using.
+                        </p>
+                    </div>
+                    <div class="col-lg-4 px-2 px-lg-2 mb-3">
+                        <h4 class="mt-3">1.
+                            Open Miro. In the side menu, select 'Templates'</h4>
+                        <p>{{"![Open Miro. In the side menu, select 'Templates'](/images/practices/remote-1.png)"| markdownify}}</p>
+                    </div>
+                    <div class="col-lg-4 px-2 px-lg-2 mb-3">
+                        <h4 class="mt-3">2. Search for Tanzu - Solution Prioritization Template and click to
+                            ADD</h4>
+                        <p>{{"![Search for the template and click ADD](/images/practices/remote-2.png)"| markdownify}}</p>
+                    </div>
+                </div>
+            </div>
+            {{ end }}
+        </div>
+    </div>
+    <div class="container py-4">
+        <div class="github-imported-sample col-9 steps">
+            {{ if .Params.readme }}
+            {{ $user_repo := (urls.Parse .Params.repo).Path }}
+            {{ $resp := getJSON (print "https://api.github.com/repos" $user_repo "/readme") }}
+            <!-- RegEx replace removes any image references which could be broken -->
+            {{ $resp.content | base64Decode | replaceRE "\\!\\[(.*)\\]\\((.*)\\)" "IMAGE" | markdownify }}
+            {{ else }}
+            {{ .Content }}
+            {{ end }}
+        </div>
```

Comment:
These lines can be removed - they are specific to the "samples" pages for when there is a readme to import.

Reply:
Removed github class and readme stuff; kept `{{ .Content }}`